setup.cfg

```diff
@@ -27,20 +27,22 @@ packages = find:
 python_requires = >=3.8
 zip_safe = False
 install_requires =
-    invenio-oaiserver>=2.2.0,<3.0.0
-    invenio-requests>=4.0.0,<5.0.0
-    invenio-search-ui>=2.4.0,<3.0.0
-    invenio-vocabularies>=3.0.0,<4.0.0
-    invenio-administration>=2.0.0,<3.0.0
+    invenio-oaiserver>=2.4.0,<3.0.0
+    invenio-requests>=4.1.3,<5.0.0
+    invenio-search-ui>=2.9.1,<3.0.0
```

Comment:
invenio-search-ui needs to be >=2.4.0, otherwise it conflicts with app-rdm

Reply:
We can try that.

Reply:
I tried all the ways here: https://pip.pypa.io/en/latest/topics/dependency-resolution/#reduce-the-number-of-versions-pip-is-trying-to-use